tests/msim/test_msim_ACTIONX.cpp

```diff
@@ -478,6 +478,30 @@ BOOST_AUTO_TEST_CASE(COMPDAT) {
         BOOST_CHECK_NO_THROW(sim.run(io, true));
     }
 }
+BOOST_AUTO_TEST_CASE(ACTIONX_OPEN_CLOSED_WELL) {
+    const auto& deck = Parser().parseFile("/home/lnebel/git_repos/opm-pyaction/opm-common/tests/msim/MSIM_ACTIONX.DATA");
```

Comment:
Tests will be run from sudirectory tests in the build directory and we should use a relative path from there. Files listed in `TEST_DATA_FILES` will be copied there So hopefully using `"msim/MSIM_ACTIONX.DATA" should work.